src/components/Inputs/InputPair.tsx

```diff
@@ -0,0 +1,88 @@
+import { cn } from '@/libs/cn';
+
+import Input from './Input';
+
+/**
+ * InputPair 컴포넌트의 props 타입
+ */
+interface InputPairProps
+  extends Omit<React.ComponentProps<typeof Input>, 'className' | 'size'> {
+  label: string;
+  size?: 'md' | 'lg';
+  labelClassName?: string;
+  className?: string;
+}
+
+/**
+ * InputPair 컴포넌트의 사이즈별 스타일 클래스
+ */
+const sizeClasses = {
+  md: {
+    label: 'text-[1.4rem]',
+    input: 'w-[30rem] h-[4.2rem]',
+  },
+  lg: {
+    label: 'text-[1.6rem]',
+    input: 'w-[40rem] h-[4.8rem]',
+  },
+};

```

Comment:
변하지 않을 상수라면 `SIZE_CLASSES`로 변경하시는걸 추천드립니다!


```suggestion
const SIZE_CLASSES = {
  md: {
    label: 'text-[1.4rem]',
    input: 'w-[30rem] h-[4.2rem]',
  },
  lg: {
    label: 'text-[1.6rem]',
    input: 'w-[40rem] h-[4.8rem]',
  },
};
```